opba-db/src/main/resources/migration/master.xml

```diff
@@ -21,5 +21,6 @@
     <include relativeToChangelogFile="true" file="migrations/0015-add-skip-psu-auth-for-verlag-bank.xml"/>
     <includeAll relativeToChangelogFile="true" path="migrations/0015-finapi-add-status-handlers" context="finapi-sandbox-banks OR finapi-prod-banks OR finapi-sandbox-test-bank"/>
     <includeAll relativeToChangelogFile="true" path="migrations/0016-finapi-add-ais-session-update" context="finapi-sandbox-banks OR finapi-prod-banks OR finapi-sandbox-test-bank"/>
+    <include relativeToChangelogFile="true" file="migrations/0016-set-start-consent-auth-with-pin-fiducia.xml"/>
```

Comment:
Please use appropriate migration number (18)

Reply:
number 15 is also used twice. I'll fix it as well